data/mods/realguns/gun.json

```diff
@@ -1,69 +1,30 @@
 [
   {
     "id": "cop_38",
+    "copy-from": "cop_38",
```

Comment:
Hi, @DangerNoodle. Your PR looks good.

The coregame COP has "range": 3, which has been removed due to the creation of .357 magnum. Using copy-from without deleting it will copy that field.